### PR TITLE
Enforce CSP

### PR DESCRIPTION
**Pull request by adamraine**
We have been experiencing minimal violations on [CSP frontend](https://csp.googleplex.com/#!/?query_mintimestamp=1619150400000&query_maxtimestamp=1620657600000&query_product=webdev&query_collectortable=EXTERNAL&query_deduplication=ACTIONABLE&f_domain=web.dev&f_version=&f_effectivedirective=script-src-elem,base-uri,script-src&f_policytype=STRICT_DYNAMIC&f_policywithunsafeeval=false).  I also did a site crawl using puppeteer looking for any CSP violations on chrome and found none. With these results, I think it is safe to enforce the CSP.

Follow up to https://github.com/GoogleChrome/web.dev/pull/5008

### CODE REVIEW

**Review by MichaelSolati — APPROVED** [review]
lgtm, @robdodson your call when you want to merge

**Comment by robdodson** [comment]
let's give it a whirl.

**Pull request merged**
